backend/test/acceptance/api.terminals.spec.js

```diff
@@ -131,7 +137,48 @@ module.exports = function info ({ agent, sandbox, k8s, auth }) {
         hostCluster: {
           kubeApiServer: `k-g.${ingressDomain}`,
           namespace: hostNamespace
-        }
+        },
+        imageHelpText: 'Dummy Image Description'
+      })
+    })
+
+    it('should reuse a terminal session', async function () {
+      const user = auth.createUser({ id, aud })
+      const bearer = await user.bearer
+
+      const containerImage = 'fooImage:0.1.2'
+
+
+      common.stub.getCloudProfiles(sandbox)
+      k8s.stub.reuseTerminal({ bearer, username, namespace, name, target, seedName, hostNamespace, containerImage })
+
+      const res = await agent
+        .post('/api/terminals')
+        .set('cookie', await user.cookie)
+        .send({
+          method: 'create',
+          params: {
+            coordinate: {
+              namespace,
+              target
+            },
+            namespace
+          }
+        })
+
+      delay(5)
```

Comment:
why do we need a fixed delay here? is there no other way? 5= 5ms?, right?

Reply:
Fixed delay is a hack. We should find something better.

Reply:
I will make a proposal